src/pages/billing/List/ProjectListItem.ts

```diff
@@ -17,66 +17,95 @@ import { BillingProjectActions } from 'src/pages/billing/List/BillingProjectActi
 import { BillingProject } from 'src/pages/billing/models/BillingProject'
 
 
-const styles = {
-  projectListItem: selected => {
+const listItemStyle = (selected, hovered) => {
+  const style = {
+    ...Style.navList.itemContainer(selected),
+    ...Style.navList.item(selected),
+    ...(selected ? { backgroundColor: colors.dark(0.1) } : {}),
+    paddingLeft: '2rem'
+  }
+  if (hovered) {
     return {
-      ...Style.navList.itemContainer(selected),
-      ...Style.navList.item(selected),
-      ...(selected ? { backgroundColor: colors.dark(0.1) } : {}),
-      paddingLeft: '3rem'
+      ...style,
+      ...Style.navList.itemHover(selected)
     }
+  } else {
+    return style
   }
 }
 
-interface ProjectListItemProps {
+export interface ProjectListItemProps {
   project: BillingProject
   loadProjects: () => void
   isActive: boolean
-  isCreatingStatus: boolean
+  isCreating: boolean
+  isDeleting: boolean
 }
 
 export const ProjectListItem = (props: ProjectListItemProps) => {
+  const [hovered, setHovered] = useState<boolean>()
+
   const { projectName, roles, status, message, cloudPlatform } = props.project
+  const viewerRoles = _.intersection(roles, _.values(billingRoles))
+  const isOwner = _.includes(billingRoles.owner, roles)
+
   // Billing projects in an error status may have UNKNOWN for the cloudPlatform.
-  const cloudContextIcon = isKnownCloudProvider(cloudPlatform) && div({ style: { display: 'flex', marginRight: '0.5rem' } }, [
+  const cloudContextIcon = isKnownCloudProvider(cloudPlatform) && span({ style: { marginRight: '0.5rem' } }, [
     h(CloudProviderIcon, { cloudProvider: cloudPlatform })
   ])
 
-  const selectableProject = () => h(Clickable, {
-    style: { ...styles.projectListItem(props.isActive), color: props.isActive ? colors.accent(1.1) : colors.accent() },
+  const projectNameElement = span({ style: { wordBreak: 'break-all', verticalAlign: 'text-top' } }, [projectName])
```

Comment:
Billing project names can be up to 30 characters. The left-hand side listing is not resizable, and anything that doesn't fit was previously just cut off (so you wouldn't see the delete icon at all).

See screenshot in description.

Reply:
I will fix this and also add a little spacing. 

![image](https://user-images.githubusercontent.com/484484/225951322-8830459e-8641-4718-aa98-4208cc93810c.png)

Reply:
Here is the greater amount of spacing: 

![image](https://user-images.githubusercontent.com/484484/225983446-e902a80e-da85-4f85-afe6-995d6d2528d9.png)

And consistent wrapping:

![image](https://user-images.githubusercontent.com/484484/225984315-2bb479ed-0b47-4a91-91b4-c365eb1a9782.png)